disk.mayhemfile

```diff
@@ -0,0 +1,7 @@
+project: ntfs-rust
```

Comment:
Tests has been renamed to testsuite.

Reply:
I removed this part in favor of using the default _/testsuite/_ in the docker image